### PR TITLE
fix(TUP-35153)add org.apache.log4j

### PR DESCRIPTION
**Pull request by jiezhang-tlnd**

### CODE REVIEW

File: org.apache.log4j/pom.xml [hunk 1]
```diff
@@ -0,0 +1,183 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.apache</groupId>
+    <artifactId>log4j</artifactId>
+    <version>1.2.19-TALEND </version>
```
Comment (bhe-talendbj) [inline org.apache.log4j/pom.xml:7]: remove trailing space

File: org.apache.log4j/pom.xml [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+                        </goals>
+                    </execution>
+                    <execution>
+                        <id>bundle-jar</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>bundle</goal>
+                        </goals>
+                        <configuration>
+                            <classifier>bundle</classifier>
```
Comment (bhe-talendbj) [inline org.apache.log4j/pom.xml:83]: remove classifier

File: org.apache.log4j/pom.xml (hunk 1 above)
Comment (nrousseau) [reply org.apache.log4j/pom.xml:7]: <groupId>ch.qos.reload4j</groupId>
            <artifactId>reload4j</artifactId>
            <version>1.2.19-TALEND</version>

Comment (nrousseau) [reply org.apache.log4j/pom.xml:7]: need this.. not org.apache.. to be consistent with what we include

Comment (jiezhang-tlnd) [reply org.apache.log4j/pom.xml:6]: ok

File: org.apache.log4j/pom.xml [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+                            <goal>manifest</goal>
+                        </goals>
+                    </execution>
+                    <execution>
+                        <id>bundle-jar</id>
+                        <phase>package</phase>
+                        <configuration>
+                            <classifier>bundle</classifier>
+                            <instructions>
+                                <Bundle-SymbolicName>ch.qos.reload4j</Bundle-SymbolicName>
```
Comment (nrousseau) [inline org.apache.log4j/pom.xml:79]: <Bundle-SymbolicName>org.apache.log4j</Bundle-SymbolicName>

**Review by bhe-talendbj — APPROVED** [review]

**Review by bhe-talendbj — APPROVED** [review]

**Review by cmeng-talend — APPROVED** [review]

**Pull request merged**